scheduler.ts

```diff
@@ -125,7 +125,7 @@ module J2ME {
              * norm    norm   0.72x
              * high    high   .1x
              */
-            currentTimeScale = -0.03103448276 * (ctx.getPriority() * ctx.runtime.priority) + 1.031034483;
+            currentTimeScale = -0.03103448276 * (ctx.priority * ctx.runtime.priority) + 1.031034483;
```

Comment:
This change by itself might be a perf win. As I recall, an earlier implementation of *getPriority* was less performant than the current version, and that affected overall perf, as this statement is executed a lot.